File: WebApp/Models/MI/MI_WHID.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class MI_WHID : JCLib.Mvc.BaseModel 
    {
        public string WH_NO { get; set; }
        public string WH_USERID { get; set; }
        public string WH_UNA { get; set; }
        public string WH_NAME { get; set; }
        public string TASK_ID { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        // =====================================
        public string WH_KIND { get; set; }

        public string ORI_TASK_ID { get; set; }

        public string INID { get; set; }
        public string WH_GRADE { get; set; }

    }
}

File: WebApp/Scripts/app/model/MI_WHID.js
Ext.define('WEBAPP.model.MI_WHID', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WH_NO', type: 'string' },        
        { name: 'WH_USERID', type: 'string' },       
        { name: 'WH_UNA', type: 'string' },   
        { name: 'WH_NAME', type: 'string' },   
        { name: 'TASK_ID', type: 'string' },       
        { name: 'WH_KIND', type: 'string' },      
        { name: 'CREATE_TIME', type: 'string' },     
        { name: 'CREATE_USER', type: 'string' },      
        { name: 'UPDATE_TIME', type: 'string' },       
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});